spec/README.md

```diff
@@ -219,83 +219,74 @@ Example (only showing one lower case diacritic):
                     "base": "u",
                     "decompose": "ue"
                 },
-                "equivalents": {
-                    "unicode": "\u00fc",
-                    "html_decimal": "&#252;",
-                    "html_hex": "&#xfc;",
-                    "encoded_uri": "%C3%BC",
-                    "html_entity": "&uuml;",
-                    "extras": {
-                        "unicode": "u\u0308",
+                "equivalents": [
+                    {
+                        "raw": "ü"
+                        "unicode": "\u00fc",
+                        "html_decimal": "&#252;",
+                        "html_hex": "&#xfc;",
+                        "encoded_uri": "%C3%BC",
+                        "html_entity": "&uuml;"
+                    },
+                    {
+                        "raw": "u¨",
+                        "unicode": "u\\u0308",
                         "html_decimal": "u&#776;",
-                        "html_hex": "u&#x308;"
+                        "html_hex": "u&#x0308;",
+                        "encoded_uri": "u%CC%88"
                     }
-                }
+                ]
             }
         }
     }
 }
 ```
 
-##### equivalents.unicode
+##### equivalents
 
-Required  
-Type: `String`
-
-Contains an escaped unicode (hex) value of the character (e.g. `\u00f6`).
-
-##### equivalents.html_decimal
-
-Required  
-Type: `String`
+Required
+Type: `Array`
 
-Contains a HTML entity in decimal format (e.g. `&#246;`).
+Contains an array of equivalents objects created through normalization.
```

Comment:
Not only, it also contains the encoded characters for the original diacritics. Am I right?